plugins/tasks/http/src/main/java/com/walmartlabs/concord/plugins/http/Configuration.java

```diff
@@ -55,6 +56,8 @@ public class Configuration {
     private final int requestTimeout;
     private final boolean ignoreErrors;
     private final String proxy;
+    private final String proxyUser;
+    private final String proxyPassword;
```

Comment:
Use `chat[]`? A good practice to prevent string interning.

Reply:
fixed

Reply:
I'm a little confused on the issue here. Why should we try to prevent interning on just these two constant strings rather than all the others (e.g. `proxy`, `keyStorePath`, `keyStorePassword`)?

Reply:
...is it just a general rule of "don't intern credentials"

Reply:
saw keyStorePassword and do not wanna change it in this PR...

there are a lot of answers why char array is better:
https://stackoverflow.com/questions/8881291/why-is-char-preferred-over-string-for-passwords